nbs/animation.ipynb

```diff
@@ -272,7 +287,7 @@
         "\n",
         "    mp4 = open(mp4_path,'rb').read()\n",
         "    data_url = \"data:video/mp4;base64,\" + b64encode(mp4).decode()\n",
-        "    display.display( display.HTML(f'<video controls loop><source src=\"{data_url}\" type=\"video/mp4\"></video>') )\n"
+        "    display.display( display.HTML(f'<video controls loop><source src=\"{data_url}\" type=\"video/mp4\"></video>') )"
```

Comment:
we should probably add a toggle to let the user affirm if they want the video to embed. I've crashed my browser accidentally trying to embed large videos in notebooks :/